flake.nix

```diff
@@ -21,7 +21,11 @@
                     inputs.holochain-flake.devShells.${system}.rustDev
                 ];
                 packages = [
```

Comment:
Temporarily modified flake to support building wasmer from source.

Reply:
Wasmer team wants to do a bunch more testing and include their changes in a v5 release which may take some time. I think we will need to keep this for now.

Reply:
Yep, fine to keep this here but maybe a comment above the block of packages to explain what they're used for?

Reply:
added an explanatory comment